.github/workflows/ci.yml

```
@@ -0,0 +1,19 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches: main
+
+jobs:
+  build:
+    runs-on: cicd-prod-03-ubuntu-prod
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          cache: yarn
+          node-version-file: '.nvmrc'
+      - run: yarn install
+      - run: yarn build
+      - run: yarn test --passWithNoTests
```

This is not a JavaScript app (yet).

```suggestion
      - run: "echo 'TODO: tests'"
```